modules/tensor_mechanics/src/postprocessors/NormalBoundaryDisplacement.C

```diff
@@ -0,0 +1,140 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "NormalBoundaryDisplacement.h"
+
+#include "libmesh/quadrature.h"
+
+registerMooseObject("TensorMechanicsApp", NormalBoundaryDisplacement);
+
+InputParameters
+NormalBoundaryDisplacement::validParams()
+{
+  InputParameters params = SidePostprocessor::validParams();
+  params.addRequiredCoupledVar("displacements", "The names of the displacement variables");
+  MooseEnum type_options("average=0 absolute_average=1 max=2 absolute_max=3", "average");
+  params.addParam<MooseEnum>("value_type", type_options, "Type of extreme value to return.");
+  params.addParam<bool>(
+      "normalize",
+      true,
```

Comment:
I would expect false by default because it's not called NormalBoundaryNormalizedDisplacement

Reply:
I'd say this parameter shouldn't even be provided. If you're asking for the average displacement, you would divide by the area, but if you're asking for a max, you wouldn't want to do that. You should just have the choice of `value_type` dictate what is done.

Reply:
Removed.